airbyte-cdk/python/build.gradle

```diff
@@ -7,29 +7,19 @@ airbytePython {
     moduleDirectory 'airbyte_cdk'
 }
 
-task generateComponentManifestClassFiles(type: Exec) {
+tasks.register('generateComponentManifestClassFiles', Exec) {
     environment 'ROOT_DIR', rootDir.absolutePath
     commandLine 'bin/generate-component-manifest-files.sh'
+}.configure {
     dependsOn ':tools:code-generator:airbyteDocker'
 }
 
-task validateSourceYamlManifest(type: Exec) {
+tasks.register('validateSourceYamlManifest', Exec) {
     environment 'ROOT_DIR', rootDir.absolutePath
     commandLine 'bin/validate-yaml-schema.sh'
 }
 
-task runLowCodeConnectorUnitTests(type: Exec) {
+tasks.register('runLowCodeConnectorUnitTests', Exec) {
     environment 'ROOT_DIR', rootDir.absolutePath
     commandLine 'bin/low-code-unit-tests.sh'
 }
-
-task runMypyOnModifiedFiles(type: Exec) {
-    environment 'ROOT_DIR', rootDir.absolutePath
-    commandLine 'bin/run-mypy-on-modified-files.sh'
-}
-
-blackFormat.dependsOn runMypyOnModifiedFiles
-isortFormat.dependsOn generateComponentManifestClassFiles
-flakeCheck.dependsOn generateComponentManifestClassFiles
-installReqs.dependsOn generateComponentManifestClassFiles
-runMypyOnModifiedFiles.dependsOn generateComponentManifestClassFiles
```

Comment:
These tasks are made redundant by `:format`. The last task shouldn't exist as a gradle task imo, instead developers should call the shell script directly if they want.